firefox_puppeteer/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Permissions(BaseLib):
+    def remove(self, host, permission):
+        with self.marionette.using_context('chrome'):
```

Comment:
This is not done by every library. I thought we wanted to have clear context switches in the test itself. I feel we should have a chat about that to find a proper solution, so we can file a mentored bug for refactoring.